### PR TITLE
fix:add missing autoprefixer

### PR DESCRIPTION
**Pull request by DanielRuf**
fix: use export
chore: ignore .cache and .idea

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "motion-ui": "~1.2.2"
   },
   "devDependencies": {
+    "autoprefixer": "^8.1.0",
```
Comment (ncoden) [inline package.json:19]: Why ? We already have https://github.com/zurb/foundation-zurb-template/blob/1755a0060d932b02e96c38784b9fc0987a2e17bc/package.json#L25

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,5 @@ npm-debug.log
 bower_components
 dist
 *.swp
+.cache
+.idea
```
Comment (ncoden) [inline .gitignore:8]: I got it for `.idea`, but why `.cache` ?

File: src/assets/js/lib/foundation-explicit-pieces.js [hunk 3]
```diff
@@ -100,4 +100,4 @@ Foundation.plugin(Tooltip, 'Tooltip');
 
 Foundation.plugin(ResponsiveAccordionTabs, 'ResponsiveAccordionTabs');
 
-module.exports = Foundation;
+export {Foundation};
```
Comment (ncoden) [inline src/assets/js/lib/foundation-explicit-pieces.js:103]: 👍

File: package.json (hunk 1 above)
Comment (DanielRuf) [reply package.json:19]: gulp-autoprefixer does not load it (anymore) even if it is installed.

File: .gitignore (hunk 2 above)
Comment (DanielRuf) [reply .gitignore:8]: That is the sass or webpack cache.

File: package.json (hunk 1 above)
Comment (DanielRuf) [reply package.json:19]: See https://github.com/sindresorhus/gulp-autoprefixer/issues/98

Comment (DanielRuf) [reply package.json:19]: But we should probably upgrade to gulp-autoprefixer 5.0.0. In general the whole Gulp ecosystem is a nightmare.

Comment (ncoden) [reply package.json:19]: From when ?
It's not like we updated `gulp-autoprefixer` recently.

Comment (ncoden) [reply package.json:19]: > In general the whole Gulp ecosystem is a nightmare.

I couldn’t agree more

Comment (DanielRuf) [reply package.json:19]: > From when ?
> It's not like we updated gulp-autoprefixer recently.

Did a fresh install and got the error that autoprefixer can not be loaded.

**Review by ncoden — APPROVED** [review]

**Pull request merged**